image.spec.in

```diff
@@ -29,6 +30,7 @@ chmod -R u+w *
 %install
 install -d -m 755 $RPM_BUILD_ROOT%containment_image_dest_dir
 cp -pr * $RPM_BUILD_ROOT%containment_image_dest_dir
+fdupes $RPM_BUILD_ROOT%containment_image_dest_dir
```

Comment:
this command without options does not take any action on duplicated files. At least that's what I could observe from manual testing. Are we sure this command is needed here?